php-checks/src/main/java/org/sonar/php/checks/utils/PhpUnitCheck.java

```diff
@@ -262,9 +270,26 @@ public void visitFunctionCall(FunctionCallTree tree) {
     super.visitFunctionCall(tree);
   }
 
-  protected boolean isTestCaseMethod(MethodDeclarationTree tree) {
+  protected boolean isTestCaseMethod(MethodDeclarationTree tree, List<String> usesNamespaces) {
```

Comment:
We should move this logic to a common place like the symbol creation. That way, we are not evaluating every check, which checks whether a method is a test method. And we would have cross-file knowledge. That way, we would avoid collecting the use statements within every PHPUnit-related check.